CMakeLists.txt

```diff
@@ -6,7 +25,11 @@ set (CMAKE_CXX_STANDARD 11) # works for CMake v3.1+
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11") #for older CMake versions
 
 message(STATUS "--WARNING-WARNING-WARNING-- generating DEBUG builds no matter what the command line says")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")
+IF (WIN32)
```

Comment:
That should be controlled via CMAKE_BUILD_TYPE

Reply:
right in principle, but I would like to leave it for now to have the merge short.